x-pack/plugins/security_solution/public/management/cypress/e2e/response_actions/response_console/scan.cy.ts

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import type { PolicyData } from '../../../../../../common/endpoint/types';
+import type { CreateAndEnrollEndpointHostResponse } from '../../../../../../scripts/endpoint/common/endpoint_host_services';
+import {
+  inputConsoleCommand,
+  openResponseConsoleFromEndpointList,
+  submitCommand,
+  // waitForCommandToBeExecuted,
+  waitForEndpointListPageToBeLoaded,
+} from '../../../tasks/response_console';
+import type { IndexedFleetEndpointPolicyResponse } from '../../../../../../common/endpoint/data_loaders/index_fleet_endpoint_policy';
+import { createAgentPolicyTask, getEndpointIntegrationVersion } from '../../../tasks/fleet';
+
+import { login } from '../../../tasks/login';
+import { enableAllPolicyProtections } from '../../../tasks/endpoint_policy';
+import { createEndpointHost } from '../../../tasks/create_endpoint_host';
+import { deleteAllLoadedEndpointData } from '../../../tasks/delete_all_endpoint_data';
+
+describe(
+  'Response console',
+  {
+    env: {
+      ftrConfig: {
+        kbnServerArgs: [
+          `--xpack.securitySolution.enableExperimental=${JSON.stringify([
+            'responseActionScanEnabled',
+          ])}`,
+        ],
+      },
+    },
+    tags: ['@ess', '@serverless'],
+  },
+  () => {
+    beforeEach(() => {
+      login();
+    });
+
+    describe('Scan operation:', () => {
+      const homeFilePath = Cypress.env('IS_CI') ? '/home/vagrant' : '/home/ubuntu';
```

Comment:
suggestion: instead of hardcoding this, maybe remove the check for `IS_CI` and just use `~/` as the path (home directory)

Reply:
Oh yeah that would work too 😄 Thanks

Reply:
so given my testing - don't make the change to `~/` because its not working

Maybe just set it to `/home` should be enough

Reply:
the `/home` path doesn't allow writing to for vagrant on CI so I changed it back for CI e3d545bf1b43683faf0291a1f63184139da5df8d